q2_dada2/plugin_setup.py

```diff
@@ -77,13 +77,13 @@
                            'samples individually, and sequences found '
                            'chimeric in a sufficient fraction of samples are '
                            'removed.'),
-        'min_parent_abundance': ('The minimum abundance of potential parents '
-                                 'of a sequence being tested as chimeric, '
-                                 'expressed as a fold-change versus '
-                                 'the abundance of the sequence being tested. '
-                                 'Values should be greater than or equal to 1 '
-                                 '(i.e. parents should be more '
-                                 'abundant than the sequence being tested).'),
+        'min_fold_parent_over_abundance': (
+            'The minimum abundance of potential parents of a sequence being '
+            'tested as chimeric, expressed as a fold-change versus the '
+            'abundance of the sequence being tested. Values should be greater '
+            'than or equal to 1 (i.e. parents should be more abundant than '
+            'the sequence being tested). This parameter has no effect if '
+            'chimera_method is "none".'),
```

Comment:
Just FYI, the plugin is already range-checking it's inputs. So what's documented is what's asserted right now.